src/styles/index.scss

```diff
@@ -1,3 +1,47 @@
+@import './utils/variables';
+@import './blocks/card';
+@import './blocks/header';
+
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+a,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  display: inline-block;
+}
```

Comment:
You've set display: inline-block for all 'a' tags. Be aware that this might affect other 'a' tags in your project in a way you might not want. Consider making this style more specific or creating a class.